test/acceptance/features/steps/servicebindingoperator.py

```diff
@@ -9,15 +9,17 @@ class Servicebindingoperator():
     name = ""
     namespace = ""
 
+    name_pattern = f"{name}.*"
```

Comment:
why we cannot use one exact value? reg ex will match against many namespaces here, is that really necessary? Why we cannot read it from env var `SBO_NAMESPACE`?

Reply:
the `name_pattern` is for name of resources, not namespace... the namespace is exact